src/chicory/broker/rabbitmq.py

```
@@ -0,0 +1,782 @@
+        self._running = False
+        self._declared_queues: set[str] = set()
+        self._declaration_lock = asyncio.Lock()
+
+    async def _get_connection(self) -> AbstractRobustConnection:
+        """Create a new RobustConnection to RabbitMQ."""
+        return await aio_pika.connect_robust(self.dsn)
+
+    async def _get_channel(self) -> AbstractChannel:
+        """Get a channel from the connection pool."""
+        if not self._connection_pool:
+            raise RuntimeError("Connection pool is not initialized")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)